addons/website/views/snippets/s_company_team.xml

```diff
@@ -4,14 +4,17 @@
 <template id="s_company_team" name="Team">
     <section class="s_company_team pt48 pb48">
         <div class="container">
+            <h3>Meet our team</h3>
+            <p class="lead">Dedicated professionals driving our success</p>
             <div class="row s_nb_column_fixed">
-                <div class="col-lg-6 pt24 pb24">
+                <div class="col-lg-6 pt32 pb32">
                     <div class="row s_col_no_resize s_col_no_bgcolor">
-                        <div class="col-lg-4 pb16 o_not_editable" contenteditable="false">
-                            <img alt="" src="/web/image/website.s_company_team_image_1" class="img-fluid rounded-circle mx-auto o_editable_media"/>
+                        <div class="col-lg-3 pb24 o_not_editable" contenteditable="false">
+                            <img alt="" src="/web/image/website.s_company_team_image_1" class="img-fluid w-lg-100 mx-lg-auto rounded-circle o_editable_media"/>
```

Comment:
nitpicking: let's not enforce the classes order convention on lines that are just partially modified, it makes reading the diff a bit more complicated ;)